code/graphics/opengl/gropengldeferred.cpp

```diff
@@ -92,7 +106,57 @@ void gr_opengl_deferred_lighting_end()
 
 	GR_DEBUG_SCOPE("Deferred lighting end");
 
+	if (Cmdline_msaa_enabled > 0) {
+		GL_state.BindFrameBuffer(Scene_framebuffer);
```

Comment:
Just for completeness' sake, adding a debug scope here would be good

```suggestion
		GR_DEBUG_SCOPE("MSAA Pass");
		GL_state.BindFrameBuffer(Scene_framebuffer);
```

Reply:
good idea